{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/config/settings/base.py

```diff
@@ -254,6 +256,8 @@ def prefixed_cookie(name):
 
 LANGUAGE_COOKIE_NAME = prefixed_cookie("language")
 
+CORS_ALLOWED_ORIGINS = []
```

Comment:
This is the default, I'm just adding it here so it's easier to override in the inheriting modules.